app/src/main/res/layout/widget_4x1.xml

```diff
@@ -54,5 +54,14 @@
 	        android:src="@drawable/content_new_holo_light"
 	        android:background="@drawable/appwidget_bg_clickable"
 	        android:contentDescription="@string/description_add_transaction_icon"/>
+
+	   <ImageButton android:id="@+id/btn_view_account"
```

Comment:
Correct. I made it show an information icon primarily so the widget would look similar in either case. I originally had the button disappear, but it looked strange when a placeholder widget was next to a normal widget.